feat(docs): add XRPL EVM Sidechain Axelar docs (#7)
ripple/docs.xrplevm.org: docs/evm-sidechain/axelar-interchain-evm-sidechain-avalanche.md

---
html: axelar-interchain-transfer-evm-sidechain-avalanche.html
blurb: Axelar interchain transfer to Avalanche Fuji.
labels:
  - Interoperability
status: not_enabled
---
# Axelar Interchain Transfer to Avalanche Fuji

> ## Using Portal

Reviewer: Unless we plan on having alternate methods, just remove this header.

You can transfer tokens from XRPL EVM sidechain to Avalanche Fuji using Axelar Portal. In this example, we will transfer 10 XRP to Avalanche Fuji as axlXRP.

> First, set XRPL EVM sidechain as the source chain and Avalanche Fuji as the destination chain.

Reviewer: ```suggestion
1. Set XRPL EVM sidechain as the source chain and Avalanche Fuji as the destination chain.
```

![portal set chains](../img/axelar-set-chains.png)

> Then, connect your wallet using Metamask for both chains.

Reviewer: ```suggestion
2. Connect your wallet using Metamask for both chains.
```

![portal connect metamask](../img/axelar-connect-metamask.png)

> Once your wallet is connected, you can start choosing the token you want to transfer and set the desired amount.

Reviewer: ```suggestion
3. Choose the token you want to transfer and set the desired amount.
```

![portal transfer](../img/axelar-set-amount.png)

> Click on `Transfer` and confirm the transaction in your wallet.

Reviewer: ```suggestion
4. Click on `Transfer` and confirm the transaction in your wallet.
```

![portal transfer](../img/axelar-sign-transaction.png)

> Then, await the transaction to be executed.

Reviewer: Delete 
```suggestion
```

> ![portal transaction executed](../img/axelar-await-transaction.png)

Reviewer: ```suggestion
```
Don't think we need to document such a small step that shows in the UI.
